libs/ledger-live-common/src/apps/formatting.ts

```diff
@@ -1,9 +1,29 @@
 import type { AppOp, InstalledItem } from "./types";
 
-export const formatSize = (size = 0, blockSize: number): string => {
-  const formatedSize =
-    size && Math.ceil((Math.ceil(size / blockSize) * blockSize) / 1024);
-  return formatedSize ? `${formatedSize}Kb` : "";
+/**
+ * Formats a byte value into its correct size in kb or mb unit taking into
+ * the device block size. It accepts a formatting function to be able to
+ * customize the rounding if needed.
+ **/
+export const formatSize = (
+  value = 0,
+  blockSize: number,
+  floor = false
+): [string, string] => {
+  const units = ["bytes", "kbUnit", "mbUnit"];
+  const k = 1024; // 1kb unit
+
+  const bytes = Math.ceil(value / blockSize) * blockSize;
+  const i = Math.floor(Math.log(bytes) / Math.log(k)) || 1; // Nb Byte units were removed from UI
+  const rawSize = bytes / Math.pow(k, i);
+  const dm = rawSize < 1 ? 1 : i > 1 ? Math.max(0, 2) : 0;
```

Comment:
not sure i understand the `Math.max(0, 2)` ?

and for the name `rawSize`: is it the size in the computed unit ? so `sizeInUnit` or something like that ?

Reply:
what does `dm` means ? 🙈